Source/ConversationV1/Models/CreateWorkspace.swift

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright IBM Corporation 2017
+ * Copyright IBM Corporation 2018
```

Comment:
These copyright date changes are going to be problematic.  We really should not be changing the date when the file has not changed.  I realize that the generator has no way to know this.  We might need to handle this in some kind of post-processing step.

Reply:
Sure, good point. I opened an issue in `arf/planning-sdk-squad` to figure out how we want to handle copyright dates.

We should probably maintain the previous copyright dates...but I do wonder if generation changes anything. In some sense, this entire file _was_ created in 2018. It just happens to be almost exactly the same!

(But I don't think that argument would really hold...)